TodoList.Web/Views/Todos/Home.cshtml

```diff
@@ -8,131 +8,198 @@
-            </td>
-        </tr>
-        }
-    </tbody>
-</table>
+            </tbody>
+        </table>
+    </div>
+    <div class="tab-pane fade" id="calendar" role="tabpanel" aria-labelledby="calendar-tab">
+        <h2>Calendar</h2>
 
-<h2>Due date</h2>
+        @{var days = 0;}
+        <table class="table table-bordered">
+            <thead class="thead-light">
+                <tr>
+                    <th colspan="7">
+                        @DateTime.Now.ToString("MMMM")
+                    </th>
+                    
+                </tr>
+                <tr>
+                    <th>Sun</th>
+                    <th>Mon</th>
+                    <th>Tues</th>
+                    <th>Wen</th>
+                    <th>Thur</th>
+                    <th>Fri</th>
+                    <th>Sat</th>
+                </tr>
+            </thead>
+            <tbody>
 
-<table class="table">
-    <thead>
-        <tr>
-            <th>
-                Update Status
-            </th>
-            <th>
-                Title
-            </th>
-            <th>
-                Done
-            </th>
-            <th>
-                Due to
-            </th>
-            <th>
-                Tags
-            </th>
-            <th></th>
-        </tr>
-    </thead>
-    <tbody>
-        @foreach (var item in Model.CloseDueToTodos)
-        {
-        <tr>
-            <td>
-                <form asp-action="UpdateDone" method="post">
-                    <input type="checkbox" class="done-checkbox" asp-for="@item.Done" />
-                    <input type="hidden" name="id" value="@item.Id" />
-                </form>
-            </td>
-            <td>
-                @item.Title
-            </td>
-            <td>
-                @(item.Done ? "Yes" : "No")
-            </td>
-            <td>
-                <span data-value="@item.DueTo" class="timeStampValueModernized">
-                    @item.DueTo
-                </span>
-            </td>
-            <td>
-                @if (item.Tags != null)
-                {
-                    @foreach (var tag in item.Tags)
+
+                <tr>
+                    @for (var i = 0; i < Model.CalendarViewModel.OffsetFromSun; i++)
                     {
-                        <a asp-action="Index" asp-route-tag="@tag" class="badge badge-light">@tag</a>
+                        <td>
+                            
+                        </td>
                     }
-                }
-            </td>
-            <td>
-                <a asp-action="Edit" asp-route-id="@item.Id" class="btn btn-secondary">Edit</a>
-                <a asp-action="Details" asp-route-id="@item.Id" class="btn btn-info">Details</a>
-                <a asp-action="Delete" asp-route-id="@item.Id" class="btn btn-danger">Delete</a>
-            </td>
-        </tr>
-        }
-    </tbody>
-</table>
+                    @while (days < Model.CalendarViewModel.NumberOfDays)
```

Comment:
I don't think that the Monthly todos are being shown in that view.

Reply:
I seem to have left out the most relevant line.  I have rewritten it slightly to allow for the possibility of multiple todos in one day.  I will push the updated view.